pkg/xds/bootstrap/template_v2.go

```diff
@@ -43,6 +43,12 @@ admin:
       port_value: {{ .AdminPort }}
 {{ end }}
 
+layered_runtime:
+  layers:
+  - name: kuma
+    static_layer:
+      envoy.restart_features.use_apple_api_for_dns_lookups: false
+
```

Comment:
v2 differs that much from v3 here? Or you just missed the DNSPort part?

Reply:
Nevermind, just saw the comment that v2 does not have the filter. Maybe we should deem v2 incompatible with the "new" DNS model and at least warn of someone enables it with v2

Reply:
oh, that's a good point. We can make this validation in bootstrap